src/library/payfine/PayFineUI.java

```diff
@@ -5,115 +5,115 @@
 public class PayFineUI {
 
 
-    private enum uI_sTaTe { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };
+    private enum PayFineUIState { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };
 
-    private PayFineControl CoNtRoL;
-    private Scanner ScAnNeR;
-    private uI_sTaTe StAtE;
+    private PayFineControl control;
+    private Scanner scanner;
+    private PayFineUIState uiState;
 
     
     public PayFineUI(PayFineControl control) {
-        this.CoNtRoL = control;
-        ScAnNeR = new Scanner(System.in);
-        StAtE = uI_sTaTe.INITIALISED;
+        this.control = control;
+        scanner = new Scanner(System.in);
+        uiState = PayFineUIState.INITIALISED;
         control.setUI(this);
     }
-    
+
     
 
-    public void RuN() {
-        DiSpLaYoUtPuT("Pay Fine Use Case UI\n");
+    public void run() {
+        displayOutput("Pay Fine Use Case UI\n");
         
         while (true) {
             
-            switch (StAtE) {
+            switch (uiState) {
             
             case READY:
-                String PaT_Str = GeTiNpUt("Swipe patron card (press <enter> to cancel): ");
-                if (PaT_Str.length() == 0) {
-                    CoNtRoL.cancel();
+                String patStr = getInput("Swipe patron card (press <enter> to cancel): ");
+                if (patStr.length() == 0) {
+                    control.cancel();
                     break;
                 }
                 try {
-                    long PAtroN_ID = Long.valueOf(PaT_Str).longValue();
-                    CoNtRoL.cardSwiped(PAtroN_ID);
+                    long patronId = Long.valueOf(patStr).longValue();
+                    control.cardSwiped(patronId);
                 }
                 catch (NumberFormatException e) {
-                    DiSpLaYoUtPuT("Invalid patronID");
+                    displayOutput("Invalid patronID");
                 }
                 break;
                 
             case PAYING:
                 double AmouNT = 0;
-                String Amt_Str = GeTiNpUt("Enter amount (<Enter> cancels) : ");
+                String Amt_Str = getInput("Enter amount (<Enter> cancels) : ");
```

Comment:
PR 45 D2 CSG#2 All variable names are to start with a lowercase letter and to be in camelBack

Reply:
[0931b27](https://github.com/elliotmitchum/ITC205/pull/45/commits/0931b27482e30530c04d653380ec0cfe35966618)